### DISCUSSION TITLE
Curso de Adobe XD | Apoyo a la comunidad

### DISCUSSION THREAD

**Discussion by CarlosSalinasRuiz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#64] [Show and tell]
Hola equipo Explorers

Les comparto un curso gratuito de Udemy sobre Adobe XD en el cual nos da toda una introducción del mundo UX | UI.

Espero les pueda servir :) y sobre todo seguir creciendo esta comunidad de desarrolladores apasionados por superarse cada vez más.

Saludos y sigamos dando lo mejor en nuestra misión FrontEnd. 

[Curso Adobe XD](https://www.udemy.com/share/101qFQ3@EnoZdFyiuyT0NFUsTEnsOhY_MxKADoR6u_Kwo0BK3MYolQ9Hxfm-Rhjs7NB83HLpXg==/)

**Comment by JonathanJRodriguez** [comment]
Hola compañero,

Muchas gracias por la información! 
Por favor recuerda poner el link en el campo que dice url para que se pueda abrir tu página, yo cometí ese mismo error hace unos días y no era accesible el link.

![imagen](https://user-images.githubusercontent.com/99201962/155250249-199e169d-8612-4ed9-9e01-5a6ed7806af1.png)

Ejemplo:

[Curso de Udemy](https://www.udemy.com/share/101qFQ3@EnoZdFyiuyT0NFUsTEnsOhY_MxKADoR6u_Kwo0BK3MYolQ9Hxfm-Rhjs7NB83HLpXg==/)

Saludos!

**Reply by CarlosSalinasRuiz** [reply]
Muchas gracias por la observación se ha corregido :)

**Comment by manuel1488** [comment]
Gracias, muy útil.
